gravity.sh

```diff
@@ -596,6 +596,10 @@ gravity_DownloadBlocklistFromUrl() {
       if [[ $(dig "${domain}" | grep "NXDOMAIN" -c) -ge 1 ]]; then
         blocked=true
       fi;;
+    "NODATA")
+      if [[ $(dig "${domain}" | grep "NOERROR" -c) -ge 1 ]] && [[ -z $(dig +noall +answer "${domain}" |awk '{print $5}') ]]; then
```

Comment:
The second test checks for the absence of an `A` record by checking if there is an answer record and if it is empty. It is very unlikely that an answer record without a resource target shows up, do you have an example for this?

Otherwise, how about this more simple way without `awk` ?

```suggestion
      if [[ $(dig "${domain}" | grep "NOERROR" -c) -ge 1 ]] && [[ -z $(dig +short "${domain}") ]]; then
```

It should do the same thing.

Reply:
You're right. My thoughts went around a few corners and made it to complicated.